pdns/common_startup.cc

```diff
@@ -176,6 +177,7 @@ void declareArguments()
   ::arg().set("cache-ttl","Seconds to store packets in the PacketCache")="20";
   ::arg().set("negquery-cache-ttl","Seconds to store negative query results in the QueryCache")="60";
   ::arg().set("query-cache-ttl","Seconds to store query results in the QueryCache")="20";
+  ::arg().set("domain-cache-ttl","Seconds to cache list of known domains")="0";
   ::arg().set("server-id", "Returned when queried for 'id.server' TXT or NSID, defaults to hostname - disabled or custom")="";
   ::arg().set("default-soa-content","Default SOA content")="a.misconfigured.dns.server.invalid hostmaster.@ 0 10800 3600 604800 3600";
   ::arg().set("default-soa-edit","Default SOA-EDIT value")="";
```

Comment:
Some of these have a space after comma and some don't

Reply:
Indeed, and I do not know what the correct style is.

Reply:
Personally, I'd vote for with. But obviously defer to someone else

Reply:
If you're not making it worse, I'll accept it. You could run the format script over it to see what it does.

Reply:
(I'd also vote for the space)

Reply:
spaced! the other occurrences should be in another PR imo